bndtools.core/src/bndtools/central/Central.java

```diff
@@ -433,12 +433,16 @@ public static void refreshPlugins() throws Exception {
         List<File> refreshedFiles = new ArrayList<File>();
         List<Refreshable> rps = getWorkspace().getPlugins(Refreshable.class);
         boolean changed = false;
+        List<RepositoryPlugin> refreshedRepos = new ArrayList<>();
         for (Refreshable rp : rps) {
             if (rp.refresh()) {
                 changed = true;
                 File root = rp.getRoot();
                 if (root != null)
                     refreshedFiles.add(root);
+                if (rp instanceof RepositoryPlugin) {
+                    refreshedRepos.add((RepositoryPlugin) rp);
```

Comment:
Couldn't we use a boolean instead of a List since we don't need the list contents?